source/install/cloud-changelog.md

```diff
@@ -3,12 +3,70 @@
 This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.
 
 Latest Mattermost Cloud releases:
+- [Release 2021-08-25](#release-2021-08-25)
 - [Release 2021-08-12](#release-2021-08-12)
 - [Release 2021-07-29](#release-2021-07-29)
 - [Release 2021-07-15](#release-2021-07-15)
 - [Release 2021-07-01](#release-2021-07-01)
 - [Release 2021-06-16](#release-2021-06-16)
-- [Release 2021-06-02](#release-2021-06-02)
+
+## Release 2021-08-25
+
+### Highlights
+
+#### Permalink Previews
+ - Added support for permalink previews for posts in Mattermost.
+
+#### Tutorial Updates
+ - Added a tip to the **Getting Started** page for downloading Desktop Apps.
+ - Added a tip for the product switcher. The tip is skipped if not applicable.
+ - Updated tutorial icons and changed text content in tutorial tips.
+ - Added a tutorial tip for new settings and status buttons.
+
+#### Branding Changes
+ - Added a new default brand theme "Denim".
+ - The existing theme names and colors, including Mattermost, Organization, Mattermost Dark, and Windows Dark have been updated to the new Sapphire, Quartz, Indigo, & Onyx theme names and colours, respectively. Anyone using the existing themes will see slightly modified theme colors after their server or workspace is upgraded. The theme variables for the existing Mattermost, Organization, Mattermost Dark, and Windows Dark themes will still be accessible in our documentation, so a custom theme can be created with these theme variables if desired. Custom themes are unaffected by this change.
+ - Updated email templates to the new branding.
+
+#### Packaging
+ - Updated in-product strings referencing E10 & E20 to new packaging.
+ - SAML has been moved from the highest level tier (Enterprise E20) to mid-level (Professional). Certificate-based authentication feature has not changed.
+
+### Improvements
+
+#### User Interface (UI)
+ - Improved typing performance when the emoji autocomplete is open.
+ - Renamed the “View image” modal to “File preview” modal.
```

Comment:
Is this actually a change users experience?

Reply:
@AshishDhama Can you help confirm?
I know that you didn't add a release note for the PR, so perhaps this is not a visible change. https://github.com/mattermost/mattermost-webapp/pull/8492

Reply:
@amyblais This isn’t a visible change we changed the name to reflect the true use case of that component. 
https://github.com/mattermost/mattermost-webapp/pull/8526 have UI changes related to the same component.